modules/oe_translation_epoetry/src/Controller/EpoetryController.php

```diff
@@ -95,7 +105,16 @@ public static function create(ContainerInterface $container) {
    */
   public function notifications(Request $request): Response {
     $callback = NotificationEndpointResolver::resolve();
-    $server_factory = new NotificationServerFactory($callback, $this->eventDispatcher, $this->loggerChannelFactory->get('oe_translation_epoetry'), new Serializer());
+    $logger = $this->loggerChannelFactory->get('oe_translation_epoetry');
+    $server_factory = new NotificationServerFactory($callback, $this->eventDispatcher, $logger, new Serializer(), $this->ticketValidation);
+    $request_type = $request->getMethod();
+    if ($request_type === 'GET') {
```

Comment:
This was an omission from before. We need to let GET requests through as they are just returning the WSDL